daml-assistant/daml-helper/src/DamlHelper/Run.hs

```diff
@@ -393,8 +393,10 @@ runInit targetFolderM = do
     currentSdkVersion <- getSdkVersion
 
     projectFiles <- listFilesRecursive targetFolder
+    let targetFolderSep = addTrailingPathSeparator targetFolder
+    let projectFilesRel = mapMaybe (stripPrefix targetFolderSep) projectFiles
```

Comment:
The `let` is unnecessary given the previous `let`


```suggestion
        projectFilesRel = mapMaybe (stripPrefix targetFolderSep) projectFiles
```

Reply:
I disagree. This `let` reduces what's in scope for `targetFolderSep`. It's so easy to accidentally write mutually recursive stuff without any intent. That's a way to break the recursion. In fact, I would encourage you to always add these `let`s in `do` blocks. :)